python/samples/getting_started/azure_functions/03_reliable_streaming/demo.http

```diff
@@ -0,0 +1,167 @@
+###
+# Reliable Streaming with Redis - Demo HTTP Requests
+#
+# Workflow:
+# 1. POST /api/agents/TravelPlanner/run - Start durable agent run (returns conversation_id)
+# 2. GET /api/agent/stream/{conversation_id} - Read chunks from Redis (SSE or plain text)
+# 3. Resume with ?cursor={id} parameter to skip already-processed messages
+#
+# Prerequisites:
+# 1. Start Redis: docker run -d --name redis -p 6379:6379 redis:latest
+# 2. Start DTS and Azurite (see README)
+# 3. Start function app: func start
+###
+
+@baseUrl = http://localhost:7071
+@agentName = TravelPlanner
+
+###
+# ===== RECOMMENDED WORKFLOW =====
+###
+
+###
+# Test 1: Start agent run (standard endpoint - triggers Redis callback)
+# @name tokyo-trip
+POST {{baseUrl}}/api/agents/{{agentName}}/run
+Content-Type: text/plain
+
+Plan a 3-day trip to Tokyo
+
+###
+# Test 2: Stream from Redis (custom endpoint with SSE format)
+# Automatically captures conversation_id from Test 1
+@tokyoConvId = {{tokyo-trip.response.body.$.conversation_id}}
+GET {{baseUrl}}/api/agent/stream/{{tokyoConvId}}
+Accept: text/event-stream
+
+###
+# Test 3: Stream in plain text format (good for terminals)
+GET {{baseUrl}}/api/agent/stream/{{tokyoConvId}}
+Accept: text/plain
+
+###
+# Test 4: Resume from a specific cursor
+# Replace {cursor_id} with actual id from SSE events
+GET {{baseUrl}}/api/agent/stream/{{tokyoConvId}}?cursor={cursor_id}
+Accept: text/event-stream
+
+###
+# ===== MORE EXAMPLES =====
+###
+
+###
+# Test 5: Weekend trip to Paris
+# @name paris-trip
+POST {{baseUrl}}/api/agents/{{agentName}}/run
+Content-Type: text/plain
+
+Plan a weekend trip to Paris with romantic restaurants and museums
+
+###
+# Test 6: Stream Paris trip
+@parisConvId = {{paris-trip.response.body.$.conversation_id}}
+GET {{baseUrl}}/api/agent/stream/{{parisConvId}}
+Accept: text/event-stream
+
+###
+# Test 7: Check agent status
```

Comment:
Other samples dont have a very verbose `demo.http` with tests and all. It might be cleaner to keep it consistent and keep this simple and easy to follow